composer.json

```diff
@@ -20,7 +20,10 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
+        "guzzlehttp/psr7": "^1.4",
```

Comment:
Please move these dependencies to `require-dev` as not everyone uses this functionality.
To ensure a compatible version is installed (of `php-http/httplug` add a conflict rule).

Reply:
So I'm not sure if this makes sense as I understand it. This doesn't require guzzle, just the PSR implementation and if someone doesn't use it but then wants to they would have to install all -dev packages in production as well. Are you sure? We could either make it a suggest and exclude the validator if the package doesn't exist in a compiler pass, or implement our own PSR-7 Request  object. Again, this package is small and only implements the PSR-7 interfaces.

Reply:
OK, no problem then but should still be moved to require-dev to not force this anyone who wants to use this library but not this specific validator.

Reply:
So I think perhaps I'll move it to suggests instead of require-dev. I think it would be more appropriate there?